broker/server.js

```diff
@@ -11,6 +11,11 @@ const headers = {
   "content-type": "application/fhir+json",
 };
 
+//Health check endpoint
+app.get("/test", (req, res) => {
+	res.send("Success!");
```

Comment:
Do we even need to show the phrase "Success" or is simply the fact that you can hit this endpoint good enough? Were you thinking that people just might not use Postman for this and work in browser?

Reply:
Also, does the endpoint just give a 500 status code if Broker isn't working?

Reply:
It's just easier to run checks like these from a browser if you don't have Postman running. Printing "Success!" makes it unambiguous which endpoint you hit.